### PR TITLE
Digits only labels are treated as a story point and render its sum

### PR DESCRIPTION
**Pull request by hiraiva**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -158,17 +160,31 @@
...
     }
   },
+  retrieveStoryPoint: function(issue) {
+    var point = 0;
+    // treat as storypoint if label's name is only digits.
+    for (var j = 0; j < issue.labels.length; j++) {
+      label = issue.labels[j];
+      if (label.name.match(/^[0-9]+$/)) {
+         point += +label.name;
+      }
```
Comment (atsfour) [inline index.html:186]: "1/2" should be also treated as story point. 
How about adding `else if (label.name === '1/2') { point += 0.5; }` ?

File: index.html [hunk 2]
```diff
@@ -239,9 +260,11 @@
     tr.append('<th colspan="' + assignees.length + '">assigned ' + data.assigned + ' / ' + data.all + '</th>')
     thead.append(tr);
 
+console.log(data);
```
Comment (atsfour) [inline index.html]: is this line for debugging?

File: index.html (hunk 1 above)
Comment (hiraiva) [reply index.html:186]: Ok, I will add the feature.

File: index.html (hunk 2 above)
Comment (hiraiva) [reply index.html]: Sorry, it's debug code.

**Pull request merged**